src/operator-config/index.ts

```diff
@@ -0,0 +1,27 @@
+import {LegMode, TransportSubmode, TripPattern} from '@atb/sdk';
+import {AUTHORITY_ID} from '@env';
+
+const currentAppAuthorityId = AUTHORITY_ID ?? 'ATB:Authority:2';
+
+export function anyLegsWithNoValidTickets(
```

Comment:
Ville her fortsatt med begrepsbruken "we sell tickets for" som gjort i `modes_we_sell_tickets_for` og `hasLegsWeCantSellTicketsFor`, synes det begrepet er enklere å forstå, samt at vi ikke bytter begrep i løpet av flyten.

Reply:
Enig. Fikset